module/VuFindDevTools/src/VuFindDevTools/Controller/DevtoolsController.php

```diff
@@ -78,7 +78,7 @@ protected function getQueryBuilder($id)
      */
     public function deminifyAction()
     {
-        $min = trim($this->params()->fromPost('min'));
+        $min = trim(($this->params()->fromPost('min')) ? : '');
```

Comment:
Thanks for catching these! Just a stylistic question: is there a reason you use `? :` instead of the `??` operator? If the goal is to avoid null values, `??` seems preferable since it is more specific. But if there's a reason to use `? :`, we typically represent that as `?:` (without the space) in other parts of the code, so it's probably worth adjusting the style for consistency.

Reply:
...and I see now that the style issue is actually causing the build to fail. Running `composer fix` would sort everything out, but I still think `??` might be preferable to `?:` in this case.